.pre-commit-config.yaml

```diff
@@ -21,11 +21,14 @@ repos:
     hooks:
       - id: autopep8
 
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
+    hooks:
+      - id: ruff
+
   - repo: https://github.com/PyCQA/flake8
     rev: 7.1.1
     hooks:
-      - id: flake8
-        # use config from setup.cfg
       - id: flake8
```

Comment:
I think it's now OK to remove flake8 against Cython code as well. We've introduced cython-lint over a year ago (#7508) and it's working perfectly.

Reply:
Resolved in f62318a866963b3501e7c3f67b0917645df547c7 .